apps/web/src/lib/tb.ts

```diff
@@ -30,6 +33,25 @@ export async function getResponseListData(
   return;
 }
 
+export async function getResponseListDataCount(
```

Comment:
The Point of this function is to get the total responses in the response list, allowing the calculation of the total page count

Reply:
This is a good attempt but I think we should create an entirely new `pipe`. Otherwise, we will process too much data from tinybird and the response time will quickly blow up.

I am thinking out loud here:

The following command **224872** entries. Does it even make sense to know the maximum amount of data and therefore page count? Or would it be ok to navigate via the icons and remove the last icon that fast forwards you to the last page as well as the "X out of Y" number.

```
%
SELECT count(*)
FROM ping_response__v3
WHERE monitorId = {{ String(monitorId, 'openstatusPing') }}
```

![CleanShot 2023-10-01 at 10 09 29@2x](https://github.com/openstatusHQ/openstatus/assets/56969857/a08b1d7c-ea7f-4cf6-8e61-5f4aa1de942e)

Reply:
@mxkaske I actually thought about creating a new pipe just for the count. It's a good idea.
Umm, the page count is essential.  I mean without it how will we know the user is on his last page so we disable the next page button?

Reply:
Once the data received are less than the 'rows per page', we know that it is the last page. We can then disable the 'next page button'.

Supposedly we do have the *224.872** rows and 100 rows per page, we will still have **2.249** pages to go to. As a user of OpenStatus, and an incident occurs, I'm interested in the last pings and don't what happened before. 

I think diagrams are more important to get a better overview going to the first pings.

Reply:
@mxkaske This will work,which means there's no need to get the pages count

Reply:
@mxkaske done